packages/calcite-components/src/components/action-group/action-group.scss

```diff
@@ -3,9 +3,12 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-action-group-columns: Sets number of grid-template-columns when the `layout` property is `"grid"`.
- * @prop --calcite-action-group-gap: Sets the gap (gutters) between rows and columns when the `layout` property is `"grid"`.
- * @prop --calcite-action-group-padding: Sets the padding when the `layout` property is `"grid"`.
+ * @prop --calcite-action-background-color: Specifies the component's background color.
+ * @prop --calcite-action-group-border-color: Specifies the component's border color when used in a calcite-action-bar or calcite-action-menu.
+ * @prop --calcite-action-group-columns: Specifies the component's grid-template-columns when the `layout` property is `"grid"`.
+ * @prop --calcite-action-group-gap: Specifies the component's gap (gutters) between rows and columns and padding around the component, when the `layout` property is `"grid"`.
+ * @prop --calcite-action-group-padding: [Deprecated] Use --calcite-action-group-gap. Sets the padding when the `layout` property is `"grid"`.
```

Comment:
I think we might be safe to deprecate and not replace with "gap" as this `layout` property is deprecated and will be removed shortly: https://github.com/Esri/calcite-design-system/issues/9713

Reply:
These weren't added by me. https://github.com/Esri/calcite-design-system/pull/7763. However, when investigating the code I noticed `--calcite-action-group-padding` wasn't actually used anywhere so it's deprecated.

Reply:
@driskull is it safe to deprecate gap with the removal of layout prop? Or does that need to be defined elsewhere?

Reply:
I think we still need it. `layout` will just become an internal property to set the layout between grid, horizontal or vertical. So if we keep grid we also need the gap css var if we want to allow users to be able to customize the gap spacing.

Reply:
Could that be on Action Bar / Pad component? Seems weird to set that on an individual group if the groups themselves are spaced by a parent. Unless I'm misinterpreting the use case. We don't want gaps between Actions to be added.

Reply:
Yeah that's a good point. I think the parent can just be updated to handle the spacing.

Reply:
added a new commit which passes padding control to the bar, menu, pad components. Is that what y'all are thinking?

Reply:
This looks good to me. What do you think @macandcheese?

Reply:
Yes! This set up makes more sense to me as a user. 

Wondering if `gap` and `group` is better here than `space` and `items` - to me this is more applicable verbiage for what the property controls: `--calcite-action-bar-group-gap` and `--calcite-action-pad-group-gap` 

I also am curious about `--calcite-action-menu-items-space` - not sure this var (by any name) is necessary - do we want folks to have a gap between groups in an Action Menu? I'd think no.

Reply:
technically it's not limited to action-groups though, so I hesitate to use `-group-gap`. It's the gap between any slotted items inside the action-bar, action-pad, or action-menu so maybe `--calcite-[component]-items-gap` works?